.changeset/happy-shrimps-sniff.md

```diff
@@ -0,0 +1,24 @@
+---
+"@comet/cms-api": minor
+---
+
+Add Sentry module to simplify integration with Sentry.
+
+### Usage:
+
+```ts
+// main.ts
+
+app.use(Sentry.Handlers.requestHandler());
+app.use(Sentry.Handlers.tracingHandler());
+app.use(Sentry.Handlers.errorHandler());
+```
+
+```ts
+// app.module.ts
+
+SentryModule.forRoot({
+    dsn: "sentry_dsn_url",
+    enviroment: "dev",
+}),
+```
```

Comment:
I think this should be a Page in the docs as well

Reply:
Done https://github.com/vivid-planet/comet/pull/2334/commits/ce291b19e4d8872a96167f76e49f7faaed6fe4e4